requirements/dev.txt

```diff
@@ -453,15 +466,20 @@ django-treebeard==4.4
     #   -r requirements/ci.txt
     #   django-cms
     #   djangocms-attributes-field
-django-view-breadcrumbs==2.2.4
+django-two-factor-auth[phonenumberslite,webauthn]==1.16.0
```

Comment:
For me locally I was unable to install this (all the other deps seem fine) and my terminal reports: 
`zsh: no matches found: django-two-factor-auth[phonenumberslite,webauthn]==1.16.0
`

Reply:
Seems like this ⬇️ solved it for me (since I do not want to disabled 'globbing' systemwide): "...if you need to pass literal square brackets as an argument to a command, you either need to escape them or quote the argument"